client-src/elements/chromedash-ot-creation-page.ts

```diff
@@ -43,6 +43,18 @@ export class ChromedashOTCreationPage extends LitElement {
           transform: translate(-50%, -50%);
           height: 300px;
         }
+        .warning-div {
```

Comment:
There is already a `class="warning"` defined for the whole site that you can use instead of defining your own.  It uses an orange background with no border.  I think it would be good to use it so that things stay consistent and we can change it one place.  The colors are defined in static/css/main.css.